caikit/runtime/service_generation/serializers.py

```diff
@@ -50,6 +61,52 @@ def module_list(self) -> List[Type[ModuleBase]]:
     def request(self) -> "_RequestMessage":
         """Return the internal representation of the request message type for this RPC"""
 
+    def create_request_data_model(self, package_name: str) -> Type[DataBase]:
+        """Dynamically create data model for this class's input RPC"""
```

Comment:
🤔 
`Dynamically create data model for this RPC's request message`?

There might also be a fun edge case here where the class is created multiple times if this method is called multiple times. I don't know if that would be a problem or not, might be fun to at least throw in a test that makes sure nothing blows up, if one doesn't already exist

Reply:
(I'm guessing it's okay to do since this class would be stored in some `locals` that will be different on each invocation)